db/python/layers/ourdna/dashboard.py

```diff
@@ -15,93 +15,106 @@
 from models.models.participant import ParticipantInternal
 
 
-class OurDnaDashboardLayer(BaseLayer):
-    """Layer for analysis logic"""
+class SampleProcessMeta:
+    """Helper class to encapsulate sample metadata properties and calculations."""
 
-    def __init__(self, connection: Connection):
-        super().__init__(connection)
-
-        self.sample_layer = SampleLayer(connection)
-        self.participant_layer = ParticipantLayer(connection)
+    meta: dict
```

Comment:
Hmm, doesn't this end up as a static variable on the class? It's a different pattern to what we usually do. Can you just make this a regular class that gets constructed on input?